go/apps/api/openapi/spec/paths/chproxy/metrics/index.yaml

```
@@ -0,0 +1,36 @@
+    "400":
+      content:
+        application/json:
+          schema:
+            $ref: "../../../error/badRequestErrorResponse.yaml"
+      description: Invalid request body or malformed events
+    "529":
+      content:
+        application/json:
+          schema:
+            $ref: "../../../error/internalServerErrorResponse.yaml"
+      description: Service overloaded, unable to process events
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)